packages/web/config/chain-infos.ts

```diff
@@ -1704,7 +1704,7 @@ const chainInfos = (
           coinDenom: "GEO",
           coinMinimalDenom: "mGeo",
           coinDecimals: 6,
-          coinGeckoId: "geodb",
+          coinGeckoId: "pool:geo",
```

Comment:
although the geodb cgid does have a price, it looks like it can go stale from time-to-time, so I agree that a locally determined price feed may be more useful